### PR TITLE
Add R notebook for taxonomy

### PR DESCRIPTION
**Pull request by kheal**
Addresses #32.

Added R notebook for taxonomy. Also changed the associated Readme to point to the new notebook.

Please review that the following at the following.
https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/42/
https://nbviewer.org/github/microbiomedata/nmdc_notebooks/blob/32_R_taxon_notebook/taxonomic_dist_by_soil_layer/R/taxonomic_dist_soil_layer_R.ipynb

### CODE REVIEW

File: taxonomic_dist_by_soil_layer/R/taxonomic_dist_soil_layer_R.ipynb [hunk 1]
```diff
@@ -0,0 +1,2622 @@
+{
```
Comment (bmeluch) [inline taxonomic_dist_by_soil_layer/R/taxonomic_dist_soil_layer_R.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.        collectio = 'biosample_set', 
</pre><p>It seems like this still runs, but isn't the argument named 'collection' ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (bmeluch) [inline taxonomic_dist_by_soil_layer/R/taxonomic_dist_soil_layer_R.ipynb:1]: <p>Has this added any information? <code>biosample_df2</code> and <code>biosample_df3</code> look the same </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (bmeluch) [inline taxonomic_dist_by_soil_layer/R/taxonomic_dist_soil_layer_R.ipynb:1]: <p>Same question - I think this has effectively just renamed the column <code>library_preparation_has_output</code> to <code>processed_sample_id3</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (kheal) [reply taxonomic_dist_by_soil_layer/R/taxonomic_dist_soil_layer_R.ipynb:1]: <p>I'm surprised it still runs!  Will fix.</p>

Comment (kheal) [reply taxonomic_dist_by_soil_layer/R/taxonomic_dist_soil_layer_R.ipynb:1]: <p>Good eye.  I think we can skip that API call and simply rename the header from <code>pooling_id_has_input</code> to <code>processed_sample_id</code>   </p>

Comment (kheal) [reply taxonomic_dist_by_soil_layer/R/taxonomic_dist_soil_layer_R.ipynb:1]: <p>Yep!  Will do. Good catch.  That simplifies things significantly.</p>

Comment (kheal) [inline taxonomic_dist_by_soil_layer/R/taxonomic_dist_soil_layer_R.ipynb:1]: <p>We can remove this call too and change the column<code>extraction_has_input</code> to <code>processed_sample_id2</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

**Comment by kheal** [comment]
@bmeluch I think I've addressed all your issues.  Can you please re-review when you get a chance?

**Review by bmeluch — APPROVED** [review]
Yes looks good!

**Pull request merged**